qa/L0_jax_unittest/test.sh

```diff
@@ -6,4 +6,6 @@ set -xe
 
 : ${TE_PATH:=/opt/transformerengine}
 pytest -Wignore -v $TE_PATH/tests/jax
+
+pip install -r $TE_PATH/examples/jax/encoder/requirements.txt
```

Comment:
Right now, it will works as mnist requirements is a subset of encoder requirements.
But to be more future proff, can you also install the mnist requirements file?

Reply:
added